tests/yam/agama/agama.pm

```diff
@@ -0,0 +1,54 @@
+## Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Run interactive installation with Agama,
+# run playwright tests directly from the Live ISO.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base Yam::agama::agama_base;
+use strict;
+use warnings;
+
+use testapi qw(
+  assert_screen
+  assert_script_run
+  select_console
+  get_required_var
+  upload_logs
+  match_has_tag
+  reset_consoles
+);
+use power_action_utils 'power_action';
+use transactional 'process_reboot';
+
+sub run {
+    assert_screen(['agama-main-page', 'agama_product_selection'], 120);
+    $testapi::password = 'linux';
+    select_console 'root-console';
+
+    my $type = get_required_var('AGAMA_TEST');
+    my $cmd;
+    if ($type !~ /take_screenshots/) {
+        $cmd = 'PRODUCTNAME="' . get_required_var('AGAMA_PRODUCT') . '"';
```

Comment:
product name will no be needed after @lemon-suse 's PR.

Reply:
I think in playwright code we need clean, not only in product selection.